ConsoleCreateSpanAnnotation/Program.cs

```diff
@@ -0,0 +1,134 @@
+﻿using GeometryGym.Ifc;
+using GeometryGym.STEP;
+using System.Collections.Generic;
+
+namespace ConsoleCreateSpanAnnotation
+{
+    internal class Program
+    {
+        private static void Main(string[] args)
+        {
+            var database = new DatabaseIfc(ModelView.Ifc4X3NotAssigned);
+            database.Factory.ApplicationDeveloper = "Sebastian Esser";
+            // turn owner history off
+            database.Factory.Options.GenerateOwnerHistory = false;
+
+            // basic setup
+            var site = new IfcSite(database, "SiteA");
+            var project = new IfcProject(
+                site,
+                "SampleProject with a span annotation",
+                IfcUnitAssignment.Length.Metre
+                );
+
+            // create an annotation
+            var annotation = new IfcAnnotation(database)
+            {
+                Name = "DesignSpeed",
+            };
+            annotation.AddComment("Annotation item span-placed along the alignment");
+
+            // link annotation with site
+            var contained = new IfcRelContainedInSpatialStructure(site);
+            contained.RelatedElements.Add(annotation);
+
+            #region Alignment
+
+            var alignment = new IfcAlignment(site)
+            {
+                Name = "Basic alignment with a single horizontal segment"
+            };
+            alignment.AddComment("Generate alignment representation");
+
+            // semantic
+            var horizSegment = new IfcAlignmentHorizontalSegment(
+                new IfcCartesianPoint(database, 5, 10),
+                0,
+                0,
+                0,
+                200,
+                IfcAlignmentHorizontalSegmentTypeEnum.LINE);
+
+            // geometric representation of a single segment. It gets referenced by IfcCompositeCurve.segments and IfcAlignmentSegment.Representation
+            var curveSegment = new IfcCurveSegment(
+                 IfcTransitionCode.CONTSAMEGRADIENTSAMECURVATURE,
+                 new IfcAxis2Placement2D(new IfcCartesianPoint(database, 0, 0)),
+                 new IfcParameterValue(0),
+                 new IfcParameterValue(200),
```

Comment:
An `IfcLine` is parameterzied as [0,1]. If you want to give true length, use `IfcNonNegativeLengthMeasure`.